Wikipedia/Images.xcassets/T-large.imageset/Contents.json

```diff
@@ -2,7 +2,7 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "T-large.pdf"
+      "filename" : "T-large-2.pdf"
```

Comment:
Hmm... I think we'll want to delete the `T-large` and `T-small` from the assets catalog, then re-add them so it doesn't do this weird `-2` business.